src/sagemaker_training/mpi.py

```diff
@@ -266,7 +266,7 @@ def _wait_for_workers(self):  # type: () -> None
                     time.sleep(self._interval)
                 logger.info("Worker %s available for communication", host)
 
-    def _create_command(self):
+    def _create_command(self, env=None):
```

Comment:
is there anyway to add your code to this smddp command function?
https://github.com/aws/sagemaker-training-toolkit/blob/5c85f3f9b8399d3256d967674cbdc4b5c8f9cf40/src/sagemaker_training/mpi.py#L184

Reply:
Good point.

Reply:
The only nit is that the new functionality is not related to `_smddpmprun`, otherwise, it fits perfectly.